ERCS/erc-social_sbt.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: draft
```

Comment:
```suggestion
eip: 7853
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.